rules/scala_proto/private/ScalaProtoWorker.scala

```diff
@@ -40,12 +40,21 @@ object ScalaProtoWorker extends WorkerMain[Unit] {
     val scalaOut = namespace.get[File]("output_dir").toPath
     Files.createDirectories(scalaOut)
 
-    val params = s"--scala_out=$scalaOut" :: sources.map(_.getPath)
+    val params = List(s"--scala_out=$scalaOut")
```

Comment:
The protobuf rules have been improved [in the upstream repo](https://github.com/higherkindness/rules_scala/commit/e9f7843366472a36c48b793f0124f93721b7b081). I wonder if just pulling in the upstream changes would be sufficient to solve the issues you were having with protobuf.

Reply:
In in favor of landing this change then getting the lucid-master branch and upstream synced up, which would involve dealing with the proto stuff.